app/models/passenger.rb

```diff
@@ -9,6 +9,10 @@ class Passenger < ApplicationRecord
   scope :active, -> { where(active: true) }
   scope :inactive, -> { where(active: false) }
 
+  # TODO: Make configurable by user
+  MOBILITY_DEVICES = ['Boot', 'Crutches', 'Cane', 'Walker',
```

Comment:
Making a note so that I remember to check in with ST about this because I don't 100% remember what the logic was on this one.